template_feed/Microsoft.DotNet.Test.ProjectTemplates.9.0/content/MSTest-CSharp/Company.TestProject1.csproj

```diff
@@ -1,4 +1,4 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="MSTest.Sdk/3.3.1">
```

Comment:
I would use the `global.json` by default so users see duplication it and start to put a solution level, we could add a comment here or in the `global.json` for it too.

Reply:
I'll merge as-is and will do a follow-up PR. I'd prefer to have an option to not have the `global.json` added for each project.